device-registry/src/test/scala/org/genivi/sota/device_registry/db/DeviceRepositorySpec.scala

```diff
@@ -36,4 +36,19 @@ class DeviceRepositorySpec extends FunSuite
                      s shouldBe(false)
     }
   }
+
+  test("activated_at can be counted") {
+
+    val device = genDeviceT.sample.get.copy(deviceId = Some(genDeviceId.sample.get))
+    val createDevice = for {
+      uuid <- db.run(DeviceRepository.create(Namespaces.defaultNs, device))
+      now = Instant.now()
+      _ <- db.run(DeviceRepository.updateLastSeen(uuid, now))
+      count <- db.run(DeviceRepository.countActivatedDevices(now, now.plusSeconds(100)))
+    } yield count
```

Comment:
easier to write these without `db.run` and just db.run on the result of the for comprehension